docker-compose.yml

```diff
@@ -2,16 +2,28 @@ version: '3'
 
 services:
 
-  local_rds:
-    container_name: local_rds
+  rail_pm_rds:
+    container_name: rail_pm_rds
     image: postgres:14.4
     env_file: .env
     shm_size: '2gb'
     environment:
-      POSTGRES_DB: ${DB_NAME}
-      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_DB: ${RPM_DB_NAME}
+      POSTGRES_PASSWORD: ${RPM_DB_PASSWORD}
     ports:
-      - "5432:5432"
+      - "${RPM_DB_PORT}:5432"
+    command: ["postgres", "-c", "log_statement=all"]
+
+  metadata_rds:
+    container_name: metadata_rds
+    image: postgres:14.4
```

Comment:
The terraform module has postgres 15 set for `metadata-rds` need to sync this between the two.